src/App.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { FiMenu } from 'react-icons/fi';
+import { Route, Routes, useLocation } from 'react-router-dom';
+import Welcome from './components/Welcome';
+import HotelRooms from './components/HotelRooms';
+import ReservationsList from './components/ReservationsList';
+import DeleteHotel from './components/DeleteHotel';
+import ReserveForm from './components/ReserveForm';
+import AddHotel from './components/AddHotel';
+import SideBar from './components/SideBar';
+import Authentication from './components/Authentication';
+import Details from './components/Details';
+import { toggleNav } from './redux/sideBar/sideBarSlice';
+
+import './App.css';
+
+const App = () => {
```

Comment:
- [ ] OPTIONAL: Your overall desktop homepage design is still questionable:

Your implementation:

![image](https://github.com/Jolak5/hotel-booking-app-frontend/assets/57421823/f53321b6-a58f-4c54-860d-f034bdcf0edd)

The given design:

![image](https://github.com/Jolak5/hotel-booking-app-frontend/assets/57421823/ff7fd6a3-84e4-48bd-8fbc-3132bff6f194)

I suggest you apply some styling touch-ups before going to the final evaluation.

For instance, the items are not completely circular like in the design, and seem to be a little too big to fit.

Also, in the design, the current section selected in the navbar appears in green.

That way, your project will look more professional. 👍

Reply:
@Meltrust  Thank you we will work on it.